pushapkscript/requirements/base.txt

```diff
@@ -173,19 +178,17 @@ cycler==0.10.0 \
     --hash=sha256:1d8a5ae1ff6c5cf9b93e8811e581232ad8920aeec647c37316ceac982b08cb2d \
     --hash=sha256:cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8
     # via matplotlib
-decorator==4.4.2 \
-    --hash=sha256:41fa54c2a0cc4ba648be4fd43cff00aedf5b9465c9bf18d64325bc225f08f760 \
-    --hash=sha256:e3a62f0520172440ca0dcc823749319382e377f37f140a0b99ef45fecb84bfe7
-    # via
-    #   ipython
-    #   networkx
+decorator==5.0.9 \
```

Comment:
nothing in https://github.com/micheles/decorator/blob/master/CHANGES.md makes me worry too much.